porn.txt

```diff
@@ -1908842,9 +1908842,12 @@
 0.0.0.0 xyzmilfs.com
 0.0.0.0 xyzteens.com
 0.0.0.0 xzxxtube.com
-0.0.0.0 goldpornfilms.com 
+0.0.0.0 goldpornfilms.com
 0.0.0.0 bestandfree.com
 0.0.0.0 tubexclips.com
 0.0.0.0 lusthero.com
 0.0.0.0 teenda.com
-0.0.0.0 gfpimps.com
\ No newline at end of file
+0.0.0.0 gfpimps.com
+0.0.0.0 rule34.paheal.net
```

Comment:
https://mypdns.org/my-privacy-dns/porn-records/-/issues/1345

Reply:
@spirillen I think it's **EXTREMELY** hard to claim that this is a rip off of that issue. In that issue there are 7 examples of subdomains that it falls under. This adds 1 of those. If this was truly ripping from that source, the expectation would be to include all the domains.

However, just to verify, I have asked the original author of the issue where they found these domains at: https://github.com/blocklistproject/Lists/issues/641#issuecomment-1053724518